pymc/distributions/continuous.py

```diff
@@ -568,13 +568,13 @@ def rng_fn(
         size: Optional[Union[List[int], int]],
     ) -> np.ndarray:
         return stats.truncnorm.rvs(
-            a=(lower - mu) / sigma,
-            b=(upper - mu) / sigma,
-            loc=mu,
-            scale=sigma,
+            a=((lower - mu) / sigma).astype("float64"),
```

Comment:
Can you add an inline comment explaining why we do this and linking to the scipy issue?

Reply:
Done in [b1dabb8](https://github.com/pymc-devs/pymc/pull/7026/commits/b1dabb856b43b6841f9a03398633a78ec502e3bc)
lmk if it could be clearer / formatted differently